### ISSUE TITLE
Error in docker compose ? 

### ISSUE THREAD

**Issue by punkouter2018** [aspnetrun/run-aspnetcore-microservices#3]
docker-compose -f docker-compose.yml -f docker-compose.override.yml up –d

WARNING: The WEBAPP_STORAGE_HOME variable is not set. Defaulting to a blank string.
WARNING: The HOME variable is not set. Defaulting to a blank string.
Creating network "run-aspnetcore-microservices_default" with the default driver
ERROR: No such service: –d

**Comment by mehmetozkaya** [comment]
Did you change anything in docker-compose files ? if yes it could be the format error, please format again your yaml files and try again.
If no changes the docker could be the can identify the services, please remove all running containers and images than restart your docker and try again.

if you inspect the which microservices makes problem you can also run with build command;
docker-compose -f docker-compose.yml -f docker-compose.override.yml up –build

**Comment by punkouter2018** [comment]
No.. Can I just press F5 in visual study for it to work  ?  Ok.. ill try that other command  Sent from Mail for Windows 10 From: Mehmet ÖzkayaSent: Thursday, July 16, 2020 2:29 AMTo: aspnetrun/run-aspnetcore-microservicesCc: punkouter2020; AuthorSubject: Re: [aspnetrun/run-aspnetcore-microservices] Error in docker compose ? (#3) Did you change anything in docker-compose files ? if yes it could be the format error, please format again your yaml files and try again.If no changes the docker could be the can identify the services, please remove all running containers and images than restart your docker and try again.if you inspect the which microservices makes problem you can also run with build command;docker-compose -f docker-compose.yml -f docker-compose.override.yml up –build—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by punkouter2018** [comment]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Sdks\Microsoft.Docker.Sdk\build\Microsoft.VisualStudio.Docker.Compose.targets(384,5): error DT1001: ERROR: for ordering.api  Cannot create container for service ordering.api: Duplicate mount point: /root/.aspnet/https
9>C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Sdks\Microsoft.Docker.Sdk\build\Microsoft.VisualStudio.Docker.Compose.targets(384,5): error DT1001: If the error persists, try restarting Docker Desktop.
9>Done building project "docker-compose.dcproj" -- FAILED.

I dunno whats  going on.. i removed all running containers.

**Comment by mehmetozkaya** [comment]
In some cases docker not worked consistency. So my recommendation is stopping all containers , remove all containers, remove all images.
After that changed a docker setting - 'Switch to linux containers' (if it is windows containers) and restart the docker.
After that run the docker-compose command.
Since it is working -d mode, it will take time to up all microservices so wait for a while and check urls.
It should be work.

**Comment by KorsakVlad** [comment]
![image](https://user-images.githubusercontent.com/59870110/106288928-b39db280-6259-11eb-8110-fab3deca4fd0.png)

**Comment by mehmetozkaya** [comment]
For building you need to put 2 dash. it should be --build

**Comment by KorsakVlad** [comment]
Thank you for the answer @mehmetozkaya 
but it isn't working

![image](https://user-images.githubusercontent.com/59870110/106291698-02008080-625d-11eb-8d0a-1712828d6ed3.png)
